homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -5,19 +5,39 @@
 import logging
 
 import voluptuous as vol
+from miio import (  # pylint: disable=import-error
```

Comment:
Why is there a need to disable linting here? Is this also what other integrations are doing when they move the imports up?

Reply:
As far as I can tell this is normally only done by integrations that are commented out in the `requirements_all.txt` because they need to be built from source for example.
As this is not the case for `miio` : https://github.com/home-assistant/home-assistant/blob/dev/requirements_all.txt#L1537 I guess you should remove the pylint comment (and run pylint against the file to see what happens)

Reply:
@Bouni I've opened the source in a container, I've run `script/setup` then `hass` and after removing that pylint comment I still get `E0401: Unable to import 'miio' (import-error)` error.

Reply:
Got no answer so I guess this must be left as it is.